requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
+
+scipy
+pytest
```

Comment:
The tests also seem to have `numpy` as a requirement. I'd argue that `pytest` isn't actually a test requirement, just a requirement of the way that the Makefile has been setup.